alcs-api/src/decision/application-decision/application-decision.controller.ts

```diff
@@ -116,11 +137,35 @@ export class ApplicationDecisionController {
   @UserRoles(...ANY_AUTH_ROLE)
   async update(
     @Param('uuid') uuid: string,
-    @Body() appDecMeeting: UpdateApplicationDecisionDto,
+    @Body() updateDto: UpdateApplicationDecisionDto,
   ): Promise<ApplicationDecisionDto> {
+    if (updateDto.amendsUuid && updateDto.reconsidersUuid) {
+      throw new BadRequestException(
+        'Cannot set both an amends and reconsiders',
```

Comment:
Is it a correct wording 'amends and reconsiders'?

Reply:
Updated to 'Cannot create a Decision linked to both an amendment and a reconsideration'